portal-frontend/src/app/features/notice-of-intents/edit-submission/parcels/parcel-entry/parcel-entry.component.html

```diff
@@ -298,7 +298,7 @@ <h6 class="owner-search-option">OR</h6>
         Add new gov contact
       </button>
       <h6 class="owner-search-option" *ngIf="filteredOwners.length">OR</h6>
-      <label for="owner-search" *ngIf="filteredOwners.length">Select from Existing Government</label>
+      <label for="owner-search" *ngIf="filteredOwners.length">Select from Existing Government Contacts</label>
       <div>
         <mat-radio-group
```

Comment:
Does `<mat-radio-group>` here also need the conditional error class like application?